dacsspace/client.py

```diff
@@ -1,20 +1,36 @@
+from configparser import ConfigParser
+
+from asnake.aspace import ASpace
+
+
+class ASnakeConfigError(Exception):
+    pass
+
 
 class ArchivesSpaceClient:
     """Handles communication with ArchivesSpace."""
 
     def __init__(self):
-        # TODO: authenticate to AS
-        # TODO: set client property
-        # self.client = ASpace(...)
-        pass
+        config = ConfigParser()
+        config.read("dacsspace/local_settings.cfg")
+        self.aspace = ASpace(baseurl=config.get('ArchivesSpace', 'baseURL'),
+                             username=config.get('ArchivesSpace', 'user'),
+                             password=config.get('ArchivesSpace', 'password'))
+        self.repo = self.aspace.repositories(config.get('ArchivesSpace', 'repository'))
+        if isinstance(self.repo, dict):
+            raise ASnakeConfigError("Error getting repository: {}".format(self.repo.get("error")))
 
     def get_resources(self, published_only):
         """Returns data about resource records from AS.
-
         Args:
           published_only (boolean): Fetch only published records from AS
-
         Returns:
           resources (list): Full JSON of AS resource records
         """
-        pass
+        if published_only is True:
```

Comment:
This can be written more tersely:

```suggestion
        if published_only:
```